cogboard-webapp/src/components/Widget/styled.js

```diff
@@ -104,12 +105,15 @@ export const StyledEmptyCardHeader = styled(props => (
   }
 `;
 
-export const StyledCardContent = styled(CardContent)`
+export const StyledCardContent = styled(({ type, ...props }) => (
+  <CardContent {...props} />
+))`
   display: flex;
   flex-direction: column;
   flex: 1;
   position: relative;
   justify-content: space-between;
+  overflow-y: ${({ type }) => (type === 'ToDoListWidget' ? 'auto' : '')};
```

Comment:
What is this for?

Reply:
This is for widget scroll when we have to much items